django/conf/locale/fr_CA/formats.py

```diff
@@ -9,7 +9,7 @@
 MONTH_DAY_FORMAT = "j F"
 SHORT_DATE_FORMAT = "Y-m-d"
 SHORT_DATETIME_FORMAT = "Y-m-d H\xa0h\xa0i"
-FIRST_DAY_OF_WEEK = 1  # Lundi
+FIRST_DAY_OF_WEEK = 0  # dimanche
```

Comment:
@charettes Can you check this change?

Reply:
It seems to be ambiguous, I've seen many instances in Québec which has the majority of `fr_CA` users refer to Monday / _Lundi_ as the first day of the week but from looking at other locales and documentation online it seems that the agreement is around Sunday / _Dimanche_ instead.

👍 on chaning

Reply:
Thanks :+1: